app/javascript/packages/document-capture/components/back-button.tsx

```diff
@@ -9,7 +9,7 @@ interface BackLinkProps extends ComponentProps<typeof Button> {
 
 function BackButton({ includeBorder = false, ...props }: BackLinkProps) {
   const button = (
-    <Button isUnstyled {...props}>
+    <Button isUnstyled {...props} aria-label={t('forms.buttons.back')}>
       &#x2039; {t('forms.buttons.back')}
```

Comment:
Another option could be to `aria-hidden` the arrow character, which has the added advantage of not being as prone to the visual and screen-reader labels falling out of sync?

```suggestion
    <Button isUnstyled {...props}>
      <span aria-hidden="true">&#x2039;</span> {t('forms.buttons.back')}
```

Reply:
@aduth updated please check :)